src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax;
+
+public class Rectangle extends Quadrangle {
+    public Rectangle(String color, int width, int height) {
+        super(color, width, height);
+    }
+
+    @Override
+    public double area() {
+        return super.area();
+    }
```

Comment:
There's no need to override the `area` method if it only calls the superclass's method without any changes. You can remove this method to use the one from the superclass directly.